scripts/convert_publish.py

```diff
@@ -115,6 +110,15 @@ def convert_posts(self, query = {}):
                     "_SIZE_": 50
                 }
                 self.publish_tables.message.append(message)
+
+                if not orphan_comment:
+                    match = next((
+                            msg for msg in self.publish_tables.getCache() 
```

Comment:
This will not work. The cache stores data only on the part of posts that have not yet written to the database.